composer.json

```diff
@@ -12,15 +12,15 @@
         {"name": "Jonathan H. Wage", "email": "jonwage@gmail.com"}
     ],
     "require": {
-        "php": "^7.2",
-        "doctrine/annotations": "^1.2",
-        "doctrine/mongodb-odm": "^2.0.0",
+        "php": "^7.2 || ^8.0",
+        "doctrine/annotations": "^1.6",
+        "doctrine/mongodb-odm": "^2.2",
```

Comment:
Is this really necessary? The old `^2.0.0` would still allow 2.2 for PHP 8, but right now someone with PHP 7.2 would no longer be able to use doctrine/mongodb-odm 2.0/2.1.

Reply:
Indeed, the `^2.0.0` constraint should stay (unless there are some blockers but I don't think there'll be any)

Reply:
Thank you guys for feedback reverted back to 2.0.0